src/Apps/CollectorProfile/Routes/Saves2/Components/__tests__/SavesArtworks.jest.tsx

```diff
@@ -71,12 +79,39 @@ describe("SavesArtworks", () => {
       expect(screen.getByText(title)).toBeInTheDocument()
       expect(screen.getByText(description)).toBeInTheDocument()
     })
+
+    it("should render correct texts for non-default collection when user doesn't have any saved artworks", () => {
+      renderWithRelay({
+        Me: () => ({
+          defaultSaves: {
+            artworksCount: 0,
+          },
+          collection: {
+            default: false,
+            artworks: {
+              edges: [],
+            },
+          },
+        }),
+      })
+
+      const title = "Keep track of artworks you love"
+      const description =
+        "Select the heart on an artwork to save it or add it to a list."
+
+      expect(screen.getByText(title)).toBeInTheDocument()
+      expect(screen.getByText(description)).toBeInTheDocument()
+    })
```

Comment:
**suggestion**: add test case for non-default collection when user have one or more saved artworks in the default list

Reply:
wait, but I have one for this, it might not be really visible through the diff though, its [here](https://github.com/artsy/force/pull/12127/files#diff-eecfba4debf0f62dde084c8442bb93206c74c9f73f4343e077697b735e699de2R60-R81)

Reply:
ahh, sorry. missed it

Reply:
<img width="400" src="https://media.giphy.com/media/3o6Mb9EC7mNqXl9x7y/giphy.gif" />